node/src/models/Lesson.ts

```diff
@@ -22,16 +22,29 @@ export interface UpdateLessonExpectation {
   hints: UpdateHint[];
 }
 
+export interface UpdateMediaProp {
+  name: string;
+  value: string;
+}
+
+export interface UpdateMedia {
+  url: string;
+  type: string;
+  props: UpdateMediaProp[];
+}
+
 export interface UpdateLesson {
   lessonId: string;
   name: string;
   intro: string;
   dialogCategory: string;
   question: string;
-  image: string;
+  image?: string;
```

Comment:
`media` should be optional. `image` should be removed altogether now, no? It's no longer used